pkg/webservice/dao/queue_info.go

```diff
@@ -31,3 +31,16 @@ type QueueCapacity struct {
 	UsedCapacity    string `json:"usedcapacity"`
 	AbsUsedCapacity string `json:"absusedcapacity"`
 }
+
+type PartitionQueueDAOInfo struct {
+	QueueName          string                  `json:"queuename"`
+	Status             string                  `json:"status"`
+	Partition          string                  `json:"partition"`
+	MaxResource        string                  `json:"maxResource"`
+	GuaranteedResource string                  `json:"guaranteedResource"`
+	AllocatedResource  string                  `json:"allocatedResource"`
+	IsLeaf             bool                    `json:"isLeaf"`
+	IsManaged          bool                    `json:"isManaged"`
+	Parent             string                  `json:"parent"`
+	Children           []PartitionQueueDAOInfo `json:"children"`
+}
```

Comment:
We already have defined a `QueueDAOInfo`, I think we should reuse/modify that one instead of creating a new DAO structure for the Queues.

Reply:
Comment https://github.com/apache/incubator-yunikorn-core/pull/223#discussion_r563832692 covers this too.